src/test/java/name/neuhalfen/projects/crypto/bouncycastle/openpgp/keys/callbacks/RFC4880TestKeyringsDedicatedSigningKey.java

```diff
@@ -32,23 +32,22 @@
 
   public final static String UID_EMAIL = "rfc4880@example.org";
 
-  public final static Instant EXPIRED_KEY_CREATION_TIME = ZonedDateTime
-      .parse("2018-03-25T10:55:31Z").toInstant();
+  public final static Date EXPIRED_KEY_CREATION_TIME =
+          new Date(1521975331000L);
```

Comment:
Please use a 'readable' date

Reply:
I'll have to look into how to parse a date string directly into a Date, but I will see what I can do :)

Reply:
`new SimpleDateFormat("yyyy-MM-dd HH:mm").parse("2018-12-31 23:59")` should so the trick :-)